### PR TITLE
Backend frameworks

### PR DESCRIPTION
**Pull request by VikSafiullina**
High-level description of changes:
- Added options for server-side code generation: FastAPI, Flask, Django.
- Added examples of generated servers. Refer to "__examples/backend_examples_".
- Included an example of a Flask application. See "__examples/hello-webrpc_".
- Implemented a basic GitHub workflow.

### CODE REVIEW

**Comment by VikSafiullina** [comment]
waiting for #1 to be merged

File: _examples/hello-webrpc/__pycache__/client.cpython-311.pyc
Comment (VojtechVitek) [inline _examples/hello-webrpc/__pycache__/client.cpython-311.pyc:1]: Can we add `__pycache__` into .gitignore and remove this binary from the git history?

**Review by VojtechVitek — COMMENTED** [review]
This is looking great! Well done, Viktoriia 🎉 

One minor comment.

File: _examples/hello-webrpc/__pycache__/client.cpython-311.pyc
Comment (VikSafiullina) [reply _examples/hello-webrpc/__pycache__/client.cpython-311.pyc:1]: oh, you're right, i missed this

File: _examples/backends_examples/django_example.gen.py
Comment (VojtechVitek) [inline _examples/backends_examples/django_example.gen.py:1]: call this `_examples/django`?

Comment (VikSafiullina) [reply _examples/backends_examples/django_example.gen.py:1]: meaning, remove **backends_examples** dir and rename files accordingly?

Comment (VojtechVitek) [reply _examples/backends_examples/django_example.gen.py:1]: yeah I was thinking that _examples/ wouldn't have any nested folders :)

but this is very cosmetic, feel free to ignore

**Review by VojtechVitek — APPROVED** [review]
LGTM

**Review by VojtechVitek — APPROVED** [review]
LGTM. Great work @holowcost !

**Pull request merged**